src/wallet/rpcwallet.cpp

```diff
@@ -1762,7 +1762,7 @@ static RPCHelpMan gettransaction()
 
     if (verbose) {
         UniValue decoded(UniValue::VOBJ);
-        TxToUniv(*wtx.tx, uint256(), decoded, false);
+        TxToUniv(*wtx.tx, /* hashBlock= */ uint256(), /* entry= */ decoded, /* include_hex= */ false);
```

Comment:
nit: Maybe before using named args, fix the argument name first?

```suggestion
        TxToUniv(*wtx.tx, /*block_hash=*/uint256{}, /*entry=*/decoded, /*include_hex=*/false);
```

Reply:
Added a commit, done here: be3651a. I had wanted to do this but seemed like `hashBlock` was used pretty regularly so I thought there might be some convention I wasn't aware of - I guess not so I'm happy to be able to do this